show_graph.py

```diff
@@ -0,0 +1,148 @@
+"""This file creates a graph from the metric Gcloud SQL database."""
+
+import argparse
+
+import database_helpers as db_utils
+import matplotlib.pyplot as plt
+import MySQLdb
+import numpy as np
+import shell_helpers as sh_utils
+import utils
+
+
+def DrawBarGraph(connection, table_name, y_axis_field, x_axis_values,
+                 x_axis_field):
+  """This function draws a single graph based on the mean value.
+
+  Args:
+    connection: the connection to get data from the databade
+    table_name: the name of the table to get data from
+    y_axis_field: column name in the table for the y axis
+    x_axis_values: the values in x axis to compare
+    x_axis_field: column name in the table for the x axis
+  """
+  direct_lists = list()
+  envoy_lists = list()
+
+  for x in x_axis_values:
+    condition = ("where {}=\"{}\" and"
+                 " category=\"direct\"").format(x_axis_field, x)
+    direct_list = db_utils.SingleColumnToList(db_utils.GetFieldFromTable(
+        connection, table_name, field=y_axis_field, cond=condition))
+    if not direct_list:
+      print "{} {} is not found in table for direct results.".format(
+          x_axis_field, x)
+      direct_list = [0]
+
+    direct_lists.append(direct_list)
+
+    condition = ("where {}=\"{}\" and"
+                 " category=\"envoy\"").format(x_axis_field, x)
+    envoy_list = db_utils.SingleColumnToList(db_utils.GetFieldFromTable(
+        connection, table_name, field=y_axis_field, cond=condition))
+    if not envoy_list:
+      print "{} {} is not found in table for Envoy results.".format(
+          x_axis_field, x)
+      envoy_list = [0]
```

Comment:
The code above is duplicated, it would make sense to refactor into an internal function (not at top level).